packages/relgen-core/src/services/context/remote/github.ts

```diff
@@ -494,7 +494,9 @@ export const githubContextService = (
         repo: string;
         num: number;
         excludedFiles?: Set<string>;
-        excludedContexts?: Set<'ticket' | 'file-content'>;
+        excludedContexts?: {
```

Comment:
The export of top-level functions that might have dependencies on other modules is not allowed. Please consider using a Dependency Injection (DI) pattern instead.